src/utils/hooks/useWindowDimensions.ts

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from 'react'
+
+const getWindowDimensions = () => {
```

Comment:
This hook mostly taken from online source. The Pdf viewer requires size attributes in pixels, so we need to get the current viewport width to calculate a reasonable width for the modal.